### PR TITLE
Centre images in blog posts

### PR DESCRIPTION
**Pull request by MeltySnow**
## Done

Added CSS to centre images in blog posts.

## QA
- Check out this branch.
- View page locally at http://0.0.0.0:8035/
- Ensure that all images are centred.

## Fixes
Fixes issue #19

### CODE REVIEW

**Comment by pmahnke** [comment]
@MeltySnow looks like you need to use the simple one line comment format

File: static/sass/styles.scss [hunk 1]
```diff
@@ -21,3 +21,9 @@
 @include kubeflow-news-p-navigation;
 @include kubeflow-news-p-separator;
 @include kubeflow-news-p-strip;
+
+/*Centre images in blog posts*/
```
Comment (pmahnke) [inline static/sass/styles.scss]:
```suggestion
// Centre images in blog posts
```

Comment (pmahnke) [reply static/sass/styles.scss]: try this

**Comment by MeltySnow** [comment]
Done!

**Pull request merged**